app/src/main/java/com/kickstarter/services/ApiService.java

```diff
@@ -137,6 +138,9 @@
   @GET
   Observable<Response<DiscoverEnvelope>> projects(@Url String paginationUrl);
 
+  @GET("/v1/projects/{project_id}/stats")
+  Observable<Response<ProjectStats>> projectStats(@Path("project_id") String projectId);
```

Comment:
Given the param is a string, it may be more accurate to refer to it as a `project_param` rather than a `project_id`. Am assuming we could either use the slug or pid.